engine/modules/opentelemetry/inc/com/centreon/engine/modules/opentelemetry/telegraf/nagios_converter.hh

```diff
@@ -20,6 +20,18 @@
 #define CCE_MOD_OTL_NAGIOS_CONVERTER_HH
 
 namespace com::centreon::engine::modules::opentelemetry::telegraf {
+
+/**
+ * @brief nagios converter doesn't accept config parameter so it is only used
+ * for the check_result_builder_config::converter_type::nagios_converter
+ */
+class nagios_converter_config : public check_result_builder_config {
```

Comment:
Why not continuing the change also here?

Reply:
nagios_check_result_builder_config

Reply:
pb rebase